lib/client.c

```diff
@@ -52,6 +52,23 @@ _reset_connection_parameters(SF_CONNECT *sf, cJSON *parameters,
 #define _SF_STMT_TYPE_MERGE (_SF_STMT_TYPE_DML + 0x400)
 #define _SF_STMT_TYPE_MULTI_TABLE_INSERT (_SF_STMT_TYPE_DML + 0x500)
 
+/**
+ * Checks whether the client is running in force_arrow mode.
+ *
+ * @param connection pointer to SF_CONNECT
+ * @return SF_BOOLEAN_TRUE if the connection is running in force_arrow mode, otherwise SF_BOOLEAN_FALSE
+ */
+static sf_bool is_force_arrow_mode(const SF_CONNECT *connection) {
+    if (connection->query_result_format == NULL) {
+        return SF_BOOLEAN_FALSE;
+    }
+    if (strcasecmp(connection->query_result_format, "arrow_force") == 0) {
```

Comment:
`strcasecmp` does not exist in Windows. Either use `strcmp` (std library) or `sf_strncasecmp` (in platform.h) instead